src/core/src/main/java/org/apache/jmeter/gui/action/LookAndFeelCommand.java

```diff
@@ -200,7 +200,7 @@ public LookAndFeelCommand() {
     }
 
     public static boolean isDarklafTheme() {
-        return "Darklaf".equals(UIManager.getLookAndFeel().getID()); // $NON-NLS-1$
+        return "Darklaf".equalsIgnoreCase(UIManager.getLookAndFeel().getID()); // $NON-NLS-1$
```

Comment:
Wow. This is unexpected. I wonder if we could eliminate `isDarklafTheme()` altogether. It is really surprising that `isDarklafTheme()` impacts colors.